Error while checking build/O2/fullCI for d35d38cc7f9a3d546d6c6a10c20073221d4a5c30 at 2025-01-10 15:27:
```
## sw/BUILD/O2-latest/log
c++: error: unrecognized command-line option '--rtlib=compiler-rt'
c++: error: unrecognized command-line option '--rtlib=compiler-rt'
input_line_9:1051:14: error: no template named 'bitset' in namespace 'std'
input_line_9:1056:14: error: no template named 'bitset' in namespace 'std'
input_line_9:1061:14: error: no template named 'bitset' in namespace 'std'
input_line_9:1085:19: error: no member named 'bitset' in namespace 'std'
input_line_9:1085:36: error: expected member name or ';' after declaration specifiers
input_line_9:1086:8: error: no template named 'bitset' in namespace 'std'
input_line_9:1089:8: error: no template named 'bitset' in namespace 'std'
input_line_9:1090:8: error: no template named 'bitset' in namespace 'std'
input_line_9:1093:8: error: no template named 'bitset' in namespace 'std'
input_line_9:1094:8: error: no template named 'bitset' in namespace 'std'
Error: /sw/slc8_x86-64/ROOT/v6-32-06-alice1-27/bin/rootcling: compilation failure (/sw/BUILD/0c84f7b5b820902deb0711a61850bf5700585125/O2/Detectors/ZDC/calib/G__O2ZDCCalib09a588d870_dictUmbrella.h)
ninja: build stopped: subcommand failed.
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/AliceO2/13851/d35d38cc7f9a3d546d6c6a10c20073221d4a5c30/build_O2_fullCI/pretty.html).

--- a/Detectors/DCS/include/DetectorsDCS/DeliveryType.h
+++ b/Detectors/DCS/include/DetectorsDCS/DeliveryType.h
@@ -20,6 +20,7 @@
 #define O2_DCS_DELIVERY_TYPE
 
 #include <string>
+#include <bitset>
 #include <stdexcept>
 #include "DetectorsDCS/GenericFunctions.h"
 

